CMakeLists.txt

```diff
@@ -500,6 +500,19 @@ IF(QMC_MPI)
                         "Use a different MPI library or build MVAPICH2 with --disable-registration-cache configure option.")
   ENDIF()
 
+  IF(${MPI_CXX_LIBRARY_VERSION_STRING} MATCHES "Open MPI")
+    IF(NOT MPIEXEC_PREFLAGS)
+      SET(MPIEXEC_PREFLAGS "--bind-to;none")
+    ELSE(NOT MPIEXEC_PREFLAGS)
+      IF(NOT "${MPIEXEC_PREFLAGS}" MATCHES "--bind-to")
+        MESSAGE(WARNING "Default Open MPI thread affinity may be bad for QMCPACK. "
```

Comment:
Should be more precise about what 'bad' means.  It means all threads would be bound to one core, and give very poor QMCPACK performance.